### PR TITLE
Update kuudra.md

### PR DESCRIPTION
**Pull request by localneoguy**
corrections

## Information

- [x] This PR fixes a formatting issue.
- [x] This PR corrects the information in an existing guide.
- [ ] This PR adds a new guide.

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] I am sure the guide is formatted properly.

## Licensing

- [x] I agree that my contributions shall be licensed under the [Creative Commons Attribution-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-sa/4.0/),
  and that SkyKings may distribute my contributions under the same license on their website.

### CODE REVIEW

File: guides/kuudra.md [hunk 1]
```diff
@@ -276,7 +274,12 @@ ChatTriggers: nwjnaddons, volcaddons; Forge: Skyblock Utils. Please note that th
...
+Fatal Tempo is the most consistent DPS choice. 
 
-### 8. Warden Helmet or Infernal Helmet?
+### 8. Warden Helmet v/s Infernal Terror Helmet
 A Warden Helmet is preferred. Infernal Helmet offers slightly lower damage and is more expensive, making Warden the better choice for overall effectiveness.
+
+
+Edited by - lxaoen
+Dm lxaoen on Discord for corrections!
+also Jeff is RNG carried Remember ( jk he helped me with stuffs )
```
Comment (plun1331) [inline guides/kuudra.md]: We don't put credits at the bottom of our guides, you are credited in the copyright section at the end of the guide
```suggestion
```

**Review by plun1331 — CHANGES_REQUESTED** [review]
Please actually fill out the PR template (it's just check boxes)

**Comment by localneoguy** [comment]
which one?

**Comment by localneoguy** [comment]
its not letting me click

**Review by plun1331 — APPROVED** [review]

**Pull request merged**